analysis/plotting.py

```diff
@@ -125,7 +126,7 @@ def coverage_growth_plot(self, benchmark_df, axes=None):
             palette=self._fuzzer_colors,
             ax=axes)
 
-        sns.despine(offset=_DEFAULT_SPINE_OFFSET, trim=True)
+        sns.despine(ax=axes, offset=_DEFAULT_SPINE_OFFSET)
```

Comment:
Is this change here interesting? I don't really know what it's doing.

Reply:
Yes, this is a bug fix, but i should be doing this in a separate commit. PTAL at https://github.com/google/fuzzbench/pull/331